glib/src/value.rs

```diff
@@ -1005,45 +1005,47 @@ numeric!(
     gobject_ffi::g_value_set_double
 );
 
-impl ValueType for Value {
-    type Type = Value;
+pub struct Boxed(pub Value);
```

Comment:
```suggestion
pub struct BoxedValue(pub Value);
```

And then also re-export it from `src/lib.rs`